composer.lock

```diff
@@ -1,3002 +0,0 @@
-{
```

Comment:
did you forget to commit these changes? looks like the entire file has been deleted and not updated.